webapp/components/EmotionsButtons/EmotionsButtons.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <ds-flex :gutter="{ lg: 'large' }" class="emotions-flex">
+    <div v-for="emotion in Object.keys(postsEmotionsCountByEmotion)" :key="emotion">
+      <ds-flex-item :width="{ lg: '100%' }">
+        <ds-button size="large" ghost @click="toggleEmotion(emotion)" class="emotions-buttons">
+          <img :src="iconPath(emotion)" width="53" />
+        </ds-button>
+        <ds-space margin-bottom="xx-small" />
+        <div class="emotions-mobile-space">
+          <p class="emotions-label">{{ $t(`contribution.emotions-label.${emotion}`) }}</p>
+          <ds-heading style="display: inline" tag="h3" :key="postsEmotionsCountByEmotion[emotion]">
+            {{ postsEmotionsCountByEmotion[emotion] }}x
+          </ds-heading>
+          {{ $t('contribution.emotions-label.emoted') }}
+        </div>
+      </ds-flex-item>
+    </div>
+  </ds-flex>
+</template>
+<script>
+import gql from 'graphql-tag'
+import { mapGetters } from 'vuex'
+import { postsEmotionsCountByCurrentUser } from '~/graphql/PostQuery.js'
+import PostMutations from '~/graphql/PostMutations.js'
+
+export default {
+  props: {
+    post: { type: Object, default: () => {} },
+  },
+  data() {
+    return {
+      selectedEmotions: [],
+      postsEmotionsCountByEmotion: { funny: 0, happy: 0, surprised: 0, cry: 0, angry: 0 },
+    }
+  },
+  computed: {
+    ...mapGetters({
+      currentUser: 'auth/user',
+    }),
+  },
+  created() {
+    Object.keys(this.postsEmotionsCountByEmotion).map(emotion => {
+      this.emotionsCount(emotion)
+    })
+  },
+  methods: {
+    iconPath(emotion) {
```

Comment:
That's a computed property!
...
Well I just saw you're using the plural `EmotionButtons`. Why not `EmotionButton`? Tests are guiding you how you have to set the boundaries :heart: tests.